### ISSUE TITLE
Any test or example on lidar mapping

### ISSUE THREAD

**Issue by ZorAttC** [ethz-asl/nvblox_ros1#17]
I try to run the nvblox_ros on the dataset of VLP16 and I got bad result.I run fast-lio to get the tf,and the rosbag link is [https://drive.google.com/file/d/1NGTN3aULoTMp3raF75LwMu-OUtzUx-zX/view](url).

When the lidar parameter is 
![image](https://github.com/ethz-asl/nvblox_ros1/assets/93999826/42b45e61-e045-4082-b80a-c119aaa6e33c)
I got trouble

![image](https://github.com/ethz-asl/nvblox_ros1/assets/93999826/ad86e78b-5b1b-44b1-a2d4-940203570752)

When I modify the lidar parameter to be 
![image](https://github.com/ethz-asl/nvblox_ros1/assets/93999826/ec64d399-0556-468b-ab35-d26be97aa407)
I got a result like this ,it is definitely wrong.
![image](https://github.com/ethz-asl/nvblox_ros1/assets/93999826/033f0b93-d50c-4382-bd97-82287ceb8a37)

another test:
![image](https://github.com/ethz-asl/nvblox_ros1/assets/93999826/08e54a2f-af72-406c-aecb-38f29d509fc4)
![image](https://github.com/ethz-asl/nvblox_ros1/assets/93999826/6f866ae6-d807-4815-a474-e85d5c2a6de3)
some meshes come out,but they are still noisy and the ground disappear.
Since the readme has nothing to tutor me to configure the lidar and make it work,I need help sincerely.

**Comment by YinghaoJia98** [comment]
The code of lidar parameters might include some little mistakes. At least, in the line 124 of nvblox_node.cpp, it should be 'nh_private_.param("lidar_height", lidar_height_, lidar_height_);'.

**Comment by YinghaoJia98** [comment]
> The code of lidar parameters might include some little mistakes. At least, in the line 124 of nvblox_node.cpp, it should be 'nh_private_.param("lidar_height", lidar_height_, lidar_height_);'.

And the parameters of your lidar who generates the PCL should be correct absolutely.

**Comment by ZorAttC** [comment]
> > The code of lidar parameters might include some little mistakes. At least, in the line 124 of nvblox_node.cpp, it should be 'nh_private_.param("lidar_height", lidar_height_, lidar_height_);'.
> 
> And the parameters of your lidar who generates the PCL should be correct absolutely.



> > The code of lidar parameters might include some little mistakes. At least, in the line 124 of nvblox_node.cpp, it should be 'nh_private_.param("lidar_height", lidar_height_, lidar_height_);'.
> 
> And the parameters of your lidar who generates the PCL should be correct absolutely.

yeah,nvblox is strict to the sensor parameters,so I choose vdbfusion  a substitute.

**Comment by ZorAttC** [comment]
> The code of lidar parameters might include some little mistakes. At least, in the line 124 of nvblox_node.cpp, it should be 'nh_private_.param("lidar_height", lidar_height_, lidar_height_);'.



> The code of lidar parameters might include some little mistakes. At least, in the line 124 of nvblox_node.cpp, it should be 'nh_private_.param("lidar_height", lidar_height_, lidar_height_);'.

I noticed this,but still not working :(

**Comment by YinghaoJia98** [comment]
> > The code of lidar parameters might include some little mistakes. At least, in the line 124 of nvblox_node.cpp, it should be 'nh_private_.param("lidar_height", lidar_height_, lidar_height_);'.
> 
> > The code of lidar parameters might include some little mistakes. At least, in the line 124 of nvblox_node.cpp, it should be 'nh_private_.param("lidar_height", lidar_height_, lidar_height_);'.
> 
> I noticed this,but still not working :(

I have test it by a VLP-16 in simulation. It appears to be functioning normally.
![Screenshot from 2024-07-02 21-51-52](https://github.com/ethz-asl/nvblox_ros1/assets/73349262/65705a48-989c-479f-8445-cc6251f4422c)
